Libraries/LibJS/Tests/automatic-semicolon-insertion.js

```diff
@@ -43,30 +39,32 @@ function foo() {
     1;
 var curly/* semicolon inserted here */}
 
-function baz() {
-    let counter = 0;
-    let outer;
+return foo();`;
 
-    outer:
-    for (let i = 0; i < 5; ++i) {
-        for (let j = 0; j < 5; ++j) {
-            continue // semicolon inserted here
-            outer // semicolon inserted here
-        }
-        counter++;
-    }
+    expect(source).toCompile();
+    expect(new Function(source)()).toBeUndefined();
+});
+
+test("more break and continue asi", () => {
+    const source =
+`let counter = 0;
+let outer;
 
-    return counter;
+outer:
+for (let i = 0; i < 5; ++i) {
+    for (let j = 0; j < 5; ++j) {
+        continue // semicolon inserted here
+        outer // semicolon inserted here
+    }
+    counter++;
 }
 
-try {
-    assert(foo() === undefined);
-    assert(baz() === 5);
+return counter;`;
 
-    console.log("PASS");
-} catch (e) {
-    console.log("FAIL: " + e);
-}
+    expect(source).toCompile();
+    expect(new Function(source)()).toBe(5);
```

Comment:
I'd suggest we add a shortcut for this, similar to `eval` - maybe `run()` or `evaluate()`? Could be native C++ or just `new Function(source)()`, but makes the test a bit easier to read and understand IMO.

E.g.

```suggestion
    expect(evaluate(source)).toBe(5);
```